feat(content): Warn the player if they fail or decline the Free Worlds intro (#11699)
endless-sky/endless-sky: data/human/free worlds 0 prologue.txt

> 	source "Glaze"

Reviewer: A mission won't re-offer if it's been declined, so this needs to be `repeat`able.
```suggestion
	repeat 1
	source "Glaze"
```

Reviewer: If it's just `repeat` it'll keep repeating indefinitely, and I'm not sure that was the goal.

Author: The mission was not supposed to be repeatable. The text implying so was left over from a previous version.

Reviewer: You're adding this to the `to offer` section:
```c
		or
			not "FW Recon 1: offered"
			and
				has "FW Katya 7C: done"
				not "FW Recon 1: active"
				not "FW Recon 1: done"
```
If the mission does not repeat, then there's no reason to have this, unless I'm really misunderstanding something.

Reviewer: So should that part be removed?

	to offer
		has "FW Recon 0: done"
		or
			not "FW Recon 1: offered"
			and
				has "FW Katya 7C: done"
				not "FW Recon 1: active"
				not "FW Recon 1: done"
	to fail
		has "chosen sides"
	
	on offer
		conversation
			branch deferred
				has "FW Recon 1: deferred"
			`When you walk into the spaceport bar, a man in a militia uniform flags you down, and says, "Captain <last>. One of my officers told me to keep an eye out for you. Would you be willing to do another small service for the Free Worlds?"`
			choice
				`	"Perhaps. What do you want me to do?"`
					goto matter
				`	"I only agreed to a transport job. I want no part in your rebellion."`
					decline
			label deferred
			`Walking around the spaceport, you recall that a Free Worlds officer once asked for your services on <origin>. Do you want to see if they still need your help?`
			choice

> 				`	"(Sure.)"`
> 				`	"(Not right now.)"`

Reviewer: ```suggestion
				`	(Sure.)`
				`	(Not right now.)`
```

		fleet "Large Southern Pirates" 2
	
	on abort

> 		log "Flaked off escort work from Tomek Voigt, commanding officer of the Free Worlds militia."

Author: ```suggestion
		log "Aborted some escort work from the Free Worlds."
```

		has "FW Bounty 2: done"
		not "chosen sides"
		random < 30 + 5 * "assisted free worlds" * "assisted free worlds"
	on decline
		dialog phrase "free worlds declined"
	on fail

> 		log "Told a militia officer that I wouldn't do any more bounty hunting for the Free Worlds."

Reviewer 2: This looks swapped.
```suggestion
		log "Told a militia officer that I wouldn't do any more bounty hunting for the Free Worlds."
		dialog phrase "free worlds declined"
	on fail
```

	on complete
		payment 500000
		"assisted free worlds" ++
		log "Defeated the Silverhawk, an Osprey armed with alien technology, and earned a good reputation with the Free Worlds militia by hunting down other dangerous pirates."
		dialog `The militia officer who asked you to hunt down the <npc> contacts you soon after you land. "You've done us a great service," he says. "I'll be telling the Council how helpful you were." He hands you a payment of <payment>.`

# Because these missions can also be failed by having "chosen sides", this mission handles the failure messages for FW Katya.
mission "FW Katya Failed"
	non-blocking
	landing
	invisible
	to offer
		not "chosen sides"
		or
			"FW Katya 1: failed"
			"FW Katya 1B: failed"
			"FW Katya Alt 2: failed"
			"FW Katya 2: failed"
			"FW Katya 2B: failed"
			"FW Katya 2C: failed"
			"FW Katya 3: failed"
			"FW Katya 4: failed"
			"FW Katya 5: failed"
			"FW Katya 5B: failed"
			"FW Katya 5C: failed"
			"FW Katya 6: failed"
			"FW Katya 6B: failed"
			"FW Katya 6C: failed"
			"FW Katya 7: failed"
			"FW Katya 7B: failed"
			"FW Katya 7C: failed"

> 	on accept

Reviewer 2: Any particular reason for using `on accept` over `on offer`?
```suggestion
	on offer
```

Author: So the `dialog phrase` doesn't show "accept" and "decline".

	on complete
		payment 500000
		"assisted free worlds" ++

> 		log "Defeated the Silverhawk, an Osprey armed with alien technology, and earned a good reputation with the Free Worlds militia by hunting down dangerous pirates."

Reviewer: ```suggestion
		log "Defeated the Silverhawk, an Osprey armed with alien technology, and earned a good reputation with the Free Worlds militia by hunting down other dangerous pirates."
```

			has "flagship planet: Zug"
	to fail
		has "chosen sides"
	on decline

> 		log "Turned down a mission from the Free Worlds officer who threatened me. If they want someone to help them, they can look somewhere else."

Author: Turning down this mission permanently locks out the Free Worlds campaign. The consequences of doing so should be made clear in the logbook entry.
